domain/src/main/assets/test_story_id_0.json

```diff
@@ -20,6 +22,26 @@
       "outline": "",
       "title": "Prototype Exploration",
       "acquired_skill_ids": []
+    },
+    {
+      "description": "Learning about oppia app in Second Story",
+      "exploration_id": "13",
+      "destination_node_ids": [],
+      "thumbnail_filename": "",
+      "outline_is_finalized": false,
+      "id": "node_2",
+      "prerequisite_skill_ids": [],
+      "thumbnail_bg_color": "#D68F78",
+      "exp_summary_dict": {
+        "thumbnail_bg_color": "#d68453",
+        "title": "Image Click Input Interaction",
+        "language_code": "en",
+        "objective": "Learn about oppia app via testing.",
+        "id": "13"
```

Comment:
@BenHenning if you take a look at this file format you'll understand that filename is basically the exploration id itself and system loads asset corresponding the exploration id only

Reply:
Makes sense--thanks.